File: cpp/1523.count-odd-numbers-in-an-interval-range.cpp
/*
 * @lc app=leetcode id=1523 lang=cpp
 *
 * [1523] Count Odd Numbers in an Interval Range
 */

// @lc code=start
class Solution
{
public:
    int countOdds(int low, int high)
    {
        return low % 2 ? (high - low) / 2 + 1 : (high - low + 1) / 2;
    }
};
// @lc code=end

File: py/1523.count-odd-numbers-in-an-interval-range.py
#
# @lc app=leetcode id=1523 lang=python3
#
# [1523] Count Odd Numbers in an Interval Range
#

# @lc code=start
class Solution:
    def countOdds(self, low: int, high: int) -> int:
        return (high - low) // 2 + 1 if low % 2 else (high - low + 1) // 2
# @lc code=end
